packages/config/src/handlers/locations/offices.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { GATEWAY_URL } from '@config/config/constants'
+import { ServerRoute } from '@hapi/hapi'
+import { SavedBundle, Location, SavedLocation } from '@opencrvs/commons/types'
+
+async function fetchLocations(type: 'ADMIN_STRUCTURE' | 'CRVS_OFFICE') {
+  const locationUrl = new URL(
+    `location?type=${type}&_count=0&status=active`,
+    GATEWAY_URL
```

Comment:
Internal microservice going back to gateway feels like a slight abstraction leak. Good opportunity to move location API logic from gateway to config microservice :)

Reply:
yup Pyry & I were talking about the exact same thing